packages/discord.js/typings/index.d.ts

```diff
@@ -837,6 +831,18 @@ export class DataResolver extends null {
   public static resolveGuildTemplateCode(data: GuildTemplateResolvable): string;
 }
 
+export class EnumResolvers extends null {
+  private constructor();
+  public static resolveChannelType(key: string | number): ChannelType;
+  public static resolveInteractionType(key: string | number): InteractionType;
+  public static resolveApplicationCommandType(key: string | number): ApplicationCommandType;
+  public static resolveApplicationCommandOptionType(key: string | number): ApplicationCommandOptionType;
+  public static resolveApplicationCommandPermissionType(key: string | number): ApplicationCommandPermissionType;
+  public static resolveComponentType(key: string | number): ComponentType;
+  public static resolveButtonStyle(key: string | number): ButtonStyle;
+  public static resolveMessageType(key: string | number): MessageType;
+}
```

Comment:
How about the privates?

Reply:
What about them? 👀

Reply:
Yeah not entirely sure what you're asking here.

Reply:
They’re not typed here when they should be, right?

Reply:
which private fields? There are none

Reply:
Ahhh mb the functions are outside of the class, didn’t notice